lesson_10/solid/solid_app/src/main/java/com/codedifferently/lesson10/library/MediaItem.java

```diff
@@ -0,0 +1,33 @@
+package com.codedifferently.lesson10.library;
+
+import com.codedifferently.lesson10.library.exceptions.LibraryNotSetException;
+import com.codedifferently.lesson10.library.exceptions.WrongLibraryException;
+
+public abstract class MediaItem {
+  public Library library;
+
+  abstract boolean canOrCantCheckOut();
```

Comment:
Drop the `OrCant`. It's cleaner. :D

Reply:
I agree! Done! The new text change tool made it a breeze.